src/plugin/kubearmor.go

```diff
@@ -60,7 +60,7 @@ func ConvertKnoxSystemPolicyToKubeArmorPolicy(knoxPolicies []types.KnoxSystemPol
 
 		kubePolicy.Spec = policy.Spec
 
-		if kubePolicy.Kind == "KubeArmorPolicy" {
+		if kubePolicy.Kind == "KubeArmorPolicy" && policy.Spec.Action == "Allow" {
```

Comment:
@vishnusomank Why is Action allow added as a validation check here?

Reply:
@seswarrajan without this check there was an additional file rule getting added to the hardening policies which we do not want. 
```
if kubePolicy.Kind == "KubeArmorPolicy" && policy.Spec.Action == "Allow" {
			dirRule := types.KnoxMatchDirectories{
				Dir:       types.PreConfiguredKubearmorRule,
				Recursive: true,
			}
			kubePolicy.Spec.File.MatchDirectories = append(policy.Spec.File.MatchDirectories, dirRule)
		}
```